koku/masu/api/aws_bill_cleanup.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2024 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+from collections import defaultdict
+from uuid import uuid4
+
+from django.views.decorators.cache import never_cache
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.decorators import permission_classes
+from rest_framework.decorators import renderer_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+from rest_framework.settings import api_settings
+
+from api.common import log_json
+from api.provider.models import Provider
+from masu.processor import is_customer_large
+from masu.processor.tasks import cleanup_aws_bills as cleanup_aws_bills_task
+from masu.processor.tasks import PRIORITY_QUEUE
+from masu.processor.tasks import PRIORITY_QUEUE_XL
+from masu.processor.tasks import QUEUE_LIST
+
+
+LOG = logging.getLogger(__name__)
+REPORT_DATA_KEY = "cleanup_aws_bills Task IDs"
+
+
+@never_cache
+@api_view(http_method_names=["GET"])
```

Comment:
This ought to be a GET or DELETE. The GET should list all the bills that should be deleted, then the DELETE will delete those bills.

It's unclear from this endpoint what we are really identifying as needing to be deleted.